docs/src/app/sandbox.tsx

```diff
@@ -1,154 +1,198 @@
                         <HStack>
                           <Icon
                             as={ NorthlightLogoXs }
-                            pl="4"
                             boxSize={ 16 }
-                            mt="4"
+                            viewBox="0 0 20 18"
                           />
-                          <Capitalized mt="4">Northlight</Capitalized>
+                          <Capitalized>Northlight</Capitalized>
                         </HStack>
                       </Link>
-                    </HStack>
-                    <Center w="full">
-                      <SearchComponentsBar
-                        routes={ [ routes[0], routes[1] ] }
-                      />
-                    </Center>
-                    <HStack>
-                      <IconButton
-                        variant="ghost"
-                        icon={ <Icon as={ tott ? Star01Solid : Star01Duo } /> }
-                        aria-label="dark mode"
-                        onClick={ () => setTott(!tott) }
-                      />
-                      <Link as={ ReactRouterLink } to="/guide/quick-start">
-                        <Button variant="ghost">Start</Button>
-                      </Link>
-                      <Link as={ ReactRouterLink } to="./reference">
-                        <Button variant="ghost">Reference</Button>
-                      </Link>
-                      <Link
-                        isExternal={ true }
-                        href=" https://github.com/mediatool/northlight"
-                      >
-                        <Button
-                          variant="ghost"
-                          rightIcon={ <Icon as={ LinkDuo } /> }
-                        >
-                          Contribute
-                        </Button>
-                      </Link>
-                    </HStack>
-                  </Grid>
-                  <Grid
-                    gridTemplateColumns={ pageProportions }
-                    color="text.default"
-                    w="full"
-                    position="relative"
+                    </DrawerHeader>
+                    <DrawerBody sx={ { bgColor: 'background.default' } }>
+                      <MainMenu menuItems={ routes } />
+                      <SubMenu mainRoutes={ routes } />
+                    </DrawerBody>
+                  </DrawerContent>
+                </Drawer>
+
+                <Flex
+                  pt="64px"
+                  minH="100vh"
+                  direction={ { md: 'row' } }
+                  flex={ 1 }
+                  w={ { base: '100%', md: 'auto' } }
+                  maxW={ { base: '100vw', md: 'none' } }
+                  overflowX={ { base: 'hidden', md: 'initial' } }
+                >
+                  <Box
+                    w="20rem"
+                    h="100%"
+                    px={ 2 }
+                    overflowY="auto"
+                    position="fixed"
```

Comment:
This is some nitpicking, but you probably want to use the spacing and sizing tokens instead of px and rem values, e.g. pt="64px" => pt="16" and w="20rem" => w="80". You can find them under the tokens picker